### ISSUE TITLE
bug: the custom payment amount entry has insufficient contrast

### ISSUE THREAD

**Issue by Rexogamer** [elementary/website#3880]
the custom payment amount entry on the home page has insufficient contrast after tapping on it to start typing:

![screenshot of the elementary OS website, showing the custom payment amount entry - the placeholder text is white on a light grey background](https://github.com/user-attachments/assets/f17c9284-08e7-4cf4-86b4-30fdf9c1445b)

the text should probably be black or dark grey instead of white.

I'm using Firefox Nightly 145.0 on a Pixel 8.
